include/delaunator.hpp

```diff
@@ -194,9 +193,10 @@ class Delaunator {
     double m_center_x;
     double m_center_y;
     std::size_t m_hash_size;
+    std::size_t m_edge_stack[EDGE_STACK_SIZE];
```

Comment:
BTW, is it possible to make the stack static so that you don't allocate a 1024-size array on every triangulation (even for small input)?

Reply:
It has a matrix options: (stack OR heap) X (static OR singleton);
* stack, singleton is probably not possible
* stack, static -- means we allocate array even of triangulation is never called; not sure it's better
* heap static and singleton probably will be slower then stack as it will be less compact and less likely in CPU cache;

cc @flippmoke @cleeus

Reply:
I will merge PR, but feedback is welcome; I something will pop, I will add a separate issue

Reply:
> means we allocate array even no triangulation is ever called; not sure it's better

This is what JS does, although I reduced it to 512 items (2kb). Seems small enough not to matter much when done once, and much better when you e.g. have to triangulate thousands of different small inputs.